fs/index.ts

```diff
@@ -63,3 +63,15 @@ export interface IClass {
   formatStartDate: string;
   _v?: number;
 }
+
+export interface IAnnouncements {
+  announcements: {
+    _id: string;
+    title: string;
+    description: string;
+    announcer: string;
+    date: Date;
+  }[];
+  currentPage: number;
+  totalAnnouncements: number;
```

Comment:
so I used simpler pagination but I still have the current page in the announcements state. its fine though since backend isn't sending it back, my code just parses the output and updates accordingly. just a note: this is what the local state will be, and it won't get sent to backend.